pyHS100/pyHS100.py

```diff
@@ -23,8 +23,6 @@
 _LOGGER = logging.getLogger(__name__)
 
 # switch states
-SWITCH_STATE_ON = 'on'
-SWITCH_STATE_OFF = 'off'
```

Comment:
Let's keep those and let people refer to them, just in case we decide to change the representation again.